x/bank/internal/keeper/keeper.go

```diff
@@ -209,6 +211,11 @@ func (keeper BaseViewKeeper) Codespace() sdk.CodespaceType {
 	return keeper.codespace
 }
 
+// Logger returns a module-specific logger.
+func (keeper BaseViewKeeper) Logger(ctx sdk.Context) log.Logger {
+	return ctx.Logger().With("module", "x/bank")
```

Comment:
ditto

Reply:
Hey, as per the issue #3512, I thought that the Logger interface had to be implemented for all the module Keepers. While making changes, I did notice that all the Keepers don't use a Logger, but I added them anyway because that's what I interpreted from the issue. But I'm happy to discuss more on this. @alexanderbez any thoughts?

Reply:
imho, if we're not using the Logger in a Keeper then there's no need for it :)

Reply:
@fedekunze not true. The point of the originating issue was to add this to all keepers.